test/lib/seedbank/dsl_test.rb

```diff
@@ -13,7 +13,7 @@
       let(:seed_namespace) { %w[development] }
 
       it 'returns the enviroment scope' do
-        subject.must_equal seed_namespace
+        _(subject).must_equal seed_namespace
```

Comment:
I think this change is not really necessary, what is the reason for it?

Reply:
@leanovich-sp Thanks for the review
`must_` and `wont_` will be deprecated in Miniteset 6
and it causes so many warning  
```
DEPRECATED: global use of must_raise from /Users/mikeperham/src/dalli/test/test_server.rb:144. Use _(obj).must_raise instead. This will fail in Minitest 6.

```

Reply:
tbh I don't like either and I think and we are not alone https://github.com/minitest/minitest/issues/856 😄

Reply:
I wonder maybe we should just leave old version of minitest 😞

Reply:
@leanovich-sp  I removed the weird `_()` 😄